src/Aspire.Hosting.Milvus/MilvusBuilderExtensions.cs

```diff
@@ -51,6 +57,43 @@ public static IResourceBuilder<MilvusServerResource> AddMilvus(this IDistributed
 
         var milvus = new MilvusServerResource(name, apiKeyParameter);
 
+        string? connectionString = null;
```

Comment:
Probably technically don't need the connection string here since you have the client.